js/data.js

```diff
@@ -1,8 +1,26 @@
 /* exported data */
+var previousEntries = null;
 
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+function loadData(event) {
+  previousEntries = localStorage.getItem('entries-local-storage');
+  if (previousEntries !== null) {
+    data.entries = JSON.parse(previousEntries).entries;
```

Comment:
in here, you just want to set `data` to be equal to what is in local storage. you don't need to access properties or increment values. 

The value incrementation for the `nextEntryId` should be done after the new entry is unshifted to the entries array